web/src/components/molecules/Asset/Asset/AssetBody/previewToolbar.tsx

```diff
@@ -1,12 +1,13 @@
 import Button from "@reearth-cms/components/atoms/Button";
 import Icon from "@reearth-cms/components/atoms/Icon";
 import PreviewModal from "@reearth-cms/components/molecules/Asset/Asset/AssetBody/previewModal";
-import { PreviewType } from "@reearth-cms/components/molecules/Asset/Asset/AssetBody/previewTypeSelect";
+import { PreviewType } from "@reearth-cms/gql/graphql-client-api";
```

Comment:
Same as other comment, GraphQL shouldn't be imported in anything other than organisms